bittide-shake/src/Clash/Shake/Cargo.hs

```diff
@@ -0,0 +1,31 @@
+-- SPDX-FileCopyrightText: 2023 Google LLC
+--
+-- SPDX-License-Identifier: Apache-2.0
+module Clash.Shake.Cargo where
+
+import Prelude
+
+import System.Exit
+import System.IO
+import System.Process
+
+cargoBuildFirmwareProgram ::
+  FilePath ->
+  -- ^ Directory to perform the call in
+  Bool ->
```

Comment:
Could you replace this with:

```haskell
data CargoBuildMode = Release | Debug
```

That way this:

```haskell
liftIO $ cargoBuildFirmwareProgram "firmware-binaries" True
```

Turns into a much more descriptive:

```haskell
liftIO $ cargoBuildFirmwareProgram "firmware-binaries" Release
```